app/payment/session_test.go

```diff
@@ -207,25 +207,26 @@ func Test_RespondPayChProposal(t *testing.T) {
 	t.Run("happy_accept", func(t *testing.T) {
 		accept := true
 		sessionAPI := &mocks.SessionAPI{}
-		sessionAPI.On("RespondChProposal", context.Background(), proposalID, accept).Return(nil)
+		sessionAPI.On("RespondChProposal", context.Background(), proposalID, accept).Return(openedChInfo, nil)
 
-		gotErr := payment.RespondPayChProposal(context.Background(), sessionAPI, proposalID, accept)
+		gotPayChInfo, gotErr := payment.RespondPayChProposal(context.Background(), sessionAPI, proposalID, accept)
```

Comment:
Here also something like `payChannelProposalInfo` variable to store `RespondPayChProposal()`output

Reply:
Updated in [this commit](https://github.com/hyperledger-labs/perun-node/pull/116/commits/2f759e50555fbcb313b10428ddd4ab7ff753604a).